hera_librarian/models/uploads.py

```diff
@@ -58,7 +58,7 @@ class UploadInitiationResponse(BaseModel):
     # Note that transfer_providers will be tried in  the order specified here. So
     # you should put the one that requires the most arguments _first_ (otherwise everything
     # will come out as a CoreTransferManager!)
-    transfer_providers: dict[str, Union[LocalTransferManager, CoreTransferManager]]
```

Comment:
Yeah, the problem with including CoreTransferManager is that it always tries execution, and in the case where we have CoreTransferManager as an ABC it can't be instantiated.

I think this is really just a problem with us trying to do strong typing in python...